packages/sync-actions/test/cart-discounts-sync.spec.js

```diff
@@ -6,19 +6,94 @@ describe('Cart Discounts Exports', () => {
     expect(actionGroups).toEqual(['base'])
   })
 
-  it('correctly defined base actions list', () => {
-    expect(baseActionsList).toEqual([
-      { action: 'changeIsActive', key: 'isActive' },
-      { action: 'changeName', key: 'name' },
-      { action: 'changeCartPredicate', key: 'cartPredicate' },
-      { action: 'changeSortOrder', key: 'sortOrder' },
-      { action: 'changeValue', key: 'value' },
-      { action: 'changeRequiresDiscountCode', key: 'requiresDiscountCode' },
-      { action: 'changeTarget', key: 'target' },
-      { action: 'setDescription', key: 'description' },
-      { action: 'setValidFrom', key: 'validFrom' },
-      { action: 'setValidUntil', key: 'validUntil' },
-    ])
+  describe('action list', () => {
+    it('should contain `changeIsActive` action', () => {
```

Comment:
I think we should always split up assertions on arrays and objects into `arrayContaining` or `objectContaining`. It makes tests less fragile, easier to extend and gives better output during runs.